client/dashboard/profile-wizard/steps/start/index.js

```diff
@@ -53,31 +56,51 @@ const benefits = [
 	},
 ];
 
-export default class Start extends Component {
+class Start extends Component {
 	constructor() {
 		super( ...arguments );
 
 		this.state = {
-			trackingChecked: true,
+			allowTracking: true,
 		};
 
 		this.onTrackingChange = this.onTrackingChange.bind( this );
 		this.startWizard = this.startWizard.bind( this );
 		this.skipWizard = this.skipWizard.bind( this );
 	}
 
-	skipWizard() {
-		this.props.updateProfile( { skipped: true } );
+	async skipWizard() {
+		const { addNotice, isProfileItemsError, updateProfileItems } = this.props;
+
+		await updateProfileItems( { skipped: true } );
+
+		if ( isProfileItemsError ) {
+			addNotice( {
+				status: 'error',
+				message: __( 'There was a problem updating your preferences.', 'woocommerce-admin' ),
+			} );
+		}
 	}
 
-	startWizard() {
-		// @todo This should update the settings with the tracking selection. See #2281.
-		this.props.goToNextStep();
+	async startWizard() {
+		const { addNotice, isSettingsError, updateSettings } = this.props;
+
+		const allowTracking = this.state.allowTracking ? 'yes' : 'no';
+		await updateSettings( { advanced: { woocommerce_allow_tracking: allowTracking } } );
```

Comment:
Should we also save the tracking preference if they skip the profiler? As is, we would only ever have people opt-in if they also opt in to Jetpack/WooCommerce Services.

Reply:
I would think that skipping the profiler means any options on that page won't be applied (I may not have even read the page contents if I'm looking for an obvious way to skip).

/cc @jameskoster Do you have an opinion on this?